testsuite/features/support/commonlib.rb

```diff
@@ -41,16 +41,6 @@ def compute_list_to_leave_running
   do_not_kill.uniq
 end
 
-# get registration URL
-# the URL depends on whether we use a proxy or not
-def registration_url
-  if $proxy.nil?
-    "https://#{$server.full_hostname}/XMLRPC"
-  else
-    "https://#{$proxy.full_hostname}/XMLRPC"
-  end
-end
-
```

Comment:
Out of scope?

Reply:
For commonlib, yes. It's used at only one very specific point in our code and is not reusable.

Reply:
I mean that this was of out scope of this PR. But ok, fine.